client/k8s/application/util/K8sUtil.go

```diff
@@ -437,6 +452,39 @@ func (impl K8sUtil) DeleteJob(namespace string, name string, clusterConfig *Clus
 	return nil
 }
 
+func (impl K8sUtil) GetK8sConfigAndClients(clusterConfig *ClusterConfig) (*rest.Config, *http.Client, *kubernetes.Clientset, error) {
+	restConfig, err := impl.GetRestConfigByCluster(clusterConfig)
+	if err != nil {
+		impl.logger.Errorw("error in getting rest config by cluster", "err", err, "clusterName", clusterConfig.ClusterName)
```

Comment:
either clusterConfig is never empty or this will panic

Reply:
it is never empty